src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.ts

```diff
@@ -219,6 +220,11 @@ export class CreateEditEndpointComponent
       this.urlDisabled = false;
       this.targetForm.controls.endpointUrl.setValue("");
     }
+    if (selectValue === 'google-gcr') {
```

Comment:
The above condition should be the relationship of else if, not nothing!

Reply:
no need

Reply:
fixed